scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -609,7 +610,7 @@
       a bearer token for authenticating with kubernetes
     - bearer-token-refresh-seconds: interval to refresh the bearer token
     - If we have a configuration file set, then we can select the context out of that kubeconfig file with kubeconfig-context"
-  [^String config-file base-path ^String use-google-service-account? bearer-token-refresh-seconds verifying-ssl ^String ca-cert ^String ca-cert-path kubeconfig-context]
+  [^String config-file base-path ^String use-google-service-account? bearer-token-refresh-seconds verifying-ssl ^String ca-cert ^String ca-cert-path kubeconfig-context workaround-token-refresh-bug?]
```

Comment:
Can we instead call it `use-token-refreshing-authenticator?`?

Reply:
Ok.